### ISSUE TITLE
Toggle Won't Trigger When Playing Marvel Rivals

### ISSUE THREAD

**Issue by AngelSoulTV** [lulzsun/RePlays#259]
Hello,

I've been trying to use it with Marvel Rivals that came out.

My toggle doesn't work while I have the game focused, only when I go back and focus on something else.

I've tried opening it as admin, that didn't solve it. 

I've altered the toggle keybind, that didn't solve it.

I've pressed check for updates and nothing happens so I'm assuming it's up to date.

Any insight would be helpful, thank you!

**Comment by lulzsun** [comment]
It seems that it may be related to their anticheat software. I have personally experienced this with another software when trying to use a toggle bind (Discord toggle-to-talk).

Not sure when I'd personally get to figuring out a workaround, or if there even is a workaround...

**Comment by AngelSoulTV** [comment]
Oh, I see. Okay, thank you for getting back to me!

**Comment by Bmilner88** [comment]
I discovered something on this, if you run RePlays as admin and then open Marvel Rivals, it detects and records fine.

I had heard somewhere that you have to run OBS as admin if you stream/record Rivals due to the Anticheat software (mentioned in a comment from lulzsun before)

Let me know if you want my logs or if that'd help :)

**Comment by nickelberg** [comment]
What about Battlefield 2042/Cyberpunk 2077?
I had problems in BF 2024, but I understood it could happen because of AC.
In Cyberpunk case, I don't think AC is the issue.

So why my F9="Toggle Recording" bind is not working?

Anyone with similar issue?
My system is recently installed Windows 11 24H2 (in November 2042)

**Comment by Bmilner88** [comment]
> What about Battlefield 2042/Cyberpunk 2077? I had problems in BF 2024, but I understood it could happen because of AC. In Cyberpunk case, I don't think AC is the issue.
> 
> So why my F9="Toggle Recording" bind is not working?
> 
> Anyone with similar issue? My system is recently installed Windows 11 24H2 (in November 2024)

Cyberpunk appears to be working for me on latest Windows 11 update:

![Image](https://github.com/user-attachments/assets/0ed68201-b5e1-4f76-8524-1a2b31eeb4d6)

![Image](https://github.com/user-attachments/assets/687d25cc-179b-42f6-a734-550c9b84322e)

I just have an event in my Task Scheduler that automatically starts RePlays as admin so I know for sure it will work for Marvel Rivals. But just for testing purposes you should be able to just quit out of RePlays, right click on RePlays desktop icon or within the Start Menu and select **Run as Adminsitrator** :)

![Image](https://github.com/user-attachments/assets/0e5d91a3-7bf3-46fc-b53c-144121cdca98)

**Comment by Segergren** [comment]
@Bmilner88 Did you try with `Whitelist` as `Capture Mode`?

**Comment by Bmilner88** [comment]
Nah just automatic capture. I can definitely try that, do you mean without running as admin as well?

**Comment by Segergren** [comment]
> do you mean without running as admin as well?
Yes, exactly :)

**Comment by Bmilner88** [comment]
Yeah Cyberpunk still seems to be working on Whitelist mode as well :)

If you meant Marvel Rivals in Whitelist mode without admin, I did try that before with no success. The only thing that so far that seems to work on my end is running RePlays as admin.

I've noticed that even Discord hotkeys won't work while I'm tabbed into MR, so the Anticheat must be really restrictive haha

**Comment by lulzsun** [comment]
Closing this since it is stale and will be labeling it as a no-fix. There may come a time where the project gets big enough that warrants verifying with anti-cheat providers to stop this from happening, but realistically I don't see it.

Also maybe there may come a time where there will be an actual fix besides running as admin.
